### ISSUE TITLE
Request to herokuapp depricated

### ISSUE THREAD

**Issue by eafiontzi** [fxpio/composer-asset-plugin#326]
Hello,

We are using the "fxp/composer-asset-plugin": "^1.4" in our yii2 installation. About a week ago on our build we received the following error "EINVRES Request to https://bower.herokuapp.com/packages/vex failed with 502", and it keeps repeating in every build. I search the project and changed the path in the file /vendor/fxp/composer-asset-plugin/Repository/BowerRepository.php:
```
    protected function getUrl()
    {
        //20180702 Bower deprecated their registry hosted with heroku.
        //return 'https://bower.herokuapp.com/packages';
        return 'https://registry.bower.io/packages';
    }
```

But i still get the error when running bower update. Do you think I should change some other setting in the plugin?

Thank you in advance

**Comment by francoispluchino** [comment]
Update your version of this plugin. It's fixed in [v1.4.2](https://github.com/fxpio/composer-asset-plugin/releases/tag/v1.4.2).

**Comment by eafiontzi** [comment]
Thank you very much
